source/about/mattermost-for-microsoft-teams.rst

```diff
@@ -4,13 +4,45 @@ Mattermost for Microsoft Teams
 .. include:: ../_static/badges/allplans-cloud-selfhosted.rst
   :start-after: :nosearch:
 
-Mattermost for Microsoft Teams enables you to collaborate with technical & operations teams seamlessly through the Mattermost app, without leaving Microsoft Teams. This Mattermost integration includes 2 components:
+Mattermost for Microsoft Teams enables you to collaborate with technical and operations teams seamlessly through the Mattermost app, without leaving Microsoft Teams. Two integrations are available: a `connected integration <#connect-mattermost-with-microsoft-teams>`__ and an `embedded integration <#embed-mattermost-within-mattermost-teams>`__.
```

Comment:
"a connected experience" "embedded experience"

In docs, do we differentiate between integration and plugin?

Reply:
Historically, the docs referred to plugins and integrations somewhat interchangeably. Docs aim to refer to plugins as "Mattermost integrations" to reach a wider audience. The term "plugin" is reserved for technical developer documentation.